deployments/charts/kuma/templates/pre-upgrade-install-missing-crds-job.yaml

```diff
@@ -79,8 +79,6 @@ data:
       echo "/kuma/missing/crds.yaml not found or empty, it looks like there is no missing crds"
     fi
   save_missing_crds.sh: |
-    #!/usr/bin/env sh
```

Comment:
remved the shebang because we know the interpreter but we don't know the path (UBI has sh at `/usr/bin/env` but distroless at `/busybox/sh`